src/api/config/Config.ts

```diff
@@ -15,6 +15,7 @@ export default class Config implements StrykerOptions {
   timeoutFactor = 1.5;
   plugins: string[] = ['stryker-*'];
   port = 9234;
+  reporter = ['progress', 'clear-text'];
```

Comment:
In that case I think we should add a final logging statement to indicate that Stryker has completely finished, but we can do this in a different PR